src/components/LoginPanel.tsx

```diff
@@ -0,0 +1,44 @@
+import { useAuth } from 'react-oidc-context';
+import { Button, Dialog, Paper } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+const LoginDialog = styled(Dialog)(({ theme }) => ({
+  display: 'flex',
+  margin: 'auto',
+  justifyContent: 'space-between',
+  borderRadius: theme.spacing(3),
+}));
+
+const LoginPaper = styled(Paper)(() => ({
+  display: 'flex',
+  background: 'white',
+}));
+
+export interface LoginPanelProps {
+  id: string;
+  open: boolean;
+  onClose: (value: string) => void;
+}
+const LoginPanel = (props: LoginPanelProps) => {
+  const auth = useAuth();
+  const { id, open, onClose } = props;
+  const handleClose = (value: string) => {
+    onClose(value);
+  };
+
+  const dialogButton = auth.isAuthenticated ? (
```

Comment:
I think this is a little easier to read if you move this conditional rendering directly into the JSX instead of using a variable.